locales/en.yml

```diff
@@ -372,6 +372,7 @@ en:
     designer: Designer
     df_major: D♭ major
     df_minor: D♭ minor
+    digital_objects: Digital Objects
```

Comment:
For English, the pattern is to have a capital first word, lower-case second word (See "Digitized music" below).

Reply:
OK I update the file